### ISSUE TITLE
Update docs for `lerna` option in `lerna.json`

### ISSUE THREAD

**Issue by ntwb** [lerna/lerna#1237]
## Expected Behavior

The documentation regarding the `lerna` option in `lerna.json` is a little sparse and some clarification on the purpose of this value would be greatly appreciated 😄 

## Current Behavior

When running `lerna --init` the following `lerna.json` and `package.json` are created by default:

• `lerna.json`
```json
{
  "lerna": "2.8.0",
  "packages": [
    "packages/*"
  ],
  "version": "0.0.0"
}
```

• `package.json`
```json
{
	"devDependencies": {
		"lerna": "^2.8.0"
	}
}
```

Is the `2.8.0` in `lerna.json` capable of using _semantic versioning_? I ask this because the caret `^` semantic versioning `^2.8.0` is used by default in `package.json`, when Lerna v2.8.1 is released, v2.8.1 will be available for anyone to use who downloads a copy of the repo, or someone who reinstalls or updates their dependencies 

So my questions are:

1. If (when it's available) v2.8.1 is installed locally, is v2.8.0 still required because that is what is defined in `lerna.json`?

2. Can the `lerna` value in `lerna.json` support semantic versioning, i.e can I have this?
```json
{
  "lerna": "^2.8.0",
  "packages": [
    "packages/*"
  ],
  "version": "0.0.0"
}
```
 
3. This repo's `lerna.json` file doesn't contain the `lerna` value at all, is it a required field or can it be omitted or is it actually not even used?

## Possible Solution

Should `lerna --init` not use semantic versioning and omit the `^` caret for the Lerna package and define the version explicitly? i.e:

• `package.json`
```json
{
	"devDependencies": {
		"lerna": "2.8.0"
	}
}
```

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` |  all |
| `npm --version`  | n/a |
| `yarn --version` | n/a |
| `node --version` | na/ |

| OS | Version |
| --- | --- |
| NAME | VERSION |
all

**Comment by noherczeg** [comment]
@ntwb I think this will be a no issue after this gets released: #1225.

**Comment by ntwb** [comment]
Awesome, thanks, @noherczeg, closing this per the above as it has already been fixed in #1225 

FYI: #1225 is currently in the v3.0.0 milestone
